ios/ReactTestApp/ContentView.swift

```diff
@@ -327,5 +332,6 @@ extension ContentViewController {
         }
     }
 }
+#endif
```

Comment:
When we have bigger `#if` blocks, we should also add a comment here so it's somewhat easier to keep track .
```suggestion
#endif // os(iOS)
```